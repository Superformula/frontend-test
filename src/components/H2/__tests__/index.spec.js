import React from 'react';
import { mount } from 'enzyme';
import { H2 } from '../';

describe('H2', () => {
  it('should render without failing', () => {
    const wrapper = mount(<H2>All Restaurants</H2>);
    expect(wrapper).toMatchSnapshot();
  });
});
