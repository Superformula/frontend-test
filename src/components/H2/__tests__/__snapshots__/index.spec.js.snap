// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H2 should render without failing 1`] = `
.c0 {
  color: #333;
  font-weight: 300;
  font-size: 32px;
  line-height: 40px;
  margin: 32px 0;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
}

@media (min-width:758px) {
  .c0 {
    font-size: 34px;
    margin-top: 65px;
    margin-bottom: 42px;
  }
}

<H2>
  <h2
    className="c0"
  >
    All Restaurants
  </h2>
</H2>
`;
